sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -1079,6 +1105,7 @@ public partial class BlobUploadOptions
         public System.Collections.Generic.IDictionary<string, string> Metadata { get { throw null; } set { } }
         public System.IProgress<long> ProgressHandler { get { throw null; } set { } }
         public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } set { } }
+        public Azure.Storage.UploadTransactionalHashingOptions TransactionalHashingOptions { get { throw null; } set { } }
```

Comment:
Why is the class name `UploadTransactionalHashingOptions` instead of `TransactionalHashingOptions`?  In the context of `BlobProperties` this seems a little odd to me.

Reply:
There is also a `DownloadTransactionalHashingOptions` that goes on download methods and has some slightly different properties. So the class names need to be differentiated.

I also don't understand what you mean by "in the context of `BlobProperties`."

Reply:
As a users, I call `BlobClient.GetProperties()`, which has a `UploadTransactionalHashingOptions` on it.  This seem like the name could be confusing, they might be wondering why stuff associated with uploading has been permanently stored in Blob Properties.

Reply:
@seanmcc-msft I believe the confusion happens because of github collapsing sections of the diff. This isn't nested under `BlobProperties` as far as I can tell.

Maybe unstead of `Upload/DownloadTransactionalHashingOptions` we should have something that mentions where the validation occurs. I.e. `ServerSideTransactionalHashingValidationOptions` or something along these lines.

Reply:
@kasobol-msft, that is correct, I was confused by the UI.